lib/danger/request_sources/github.rb

```diff
@@ -50,9 +50,16 @@ def update_pull_request!(warnings: nil, errors: nil, messages: nil)
         # Just remove the comment, if there's nothing to say.
         delete_old_comments!
       else
+        issues = client.issue_comments(ci_source.repo_slug, ci_source.pull_request_id)
+        editable_issues = issues.reject { |issue|  issue[:body].include?("generated_by_danger") == false }
```

Comment:
Yeah, eventually, I figure that'd happen when someone builds support for non-github stuff :+1: